regression/regression_param.sh

```diff
@@ -173,11 +173,11 @@ case $regtest in
 
     ;;
 
-    netcdf_fv3_regional)
+    rrfs_enkf_conv)
 
         if [[ "$machine" = "Hera" ]]; then
-           topts[1]="0:15:00" ; popts[1]="4/4/"  ; ropts[1]="/1"
-           topts[2]="0:15:00" ; popts[2]="6/6/"  ; ropts[2]="/1"
+           topts[1]="0:05:00" ; popts[1]="40/2/"  ; ropts[1]="/1"
```

Comment:
@hu5970  If it is for 13km conus domain and 30 ensembles, does it look like memory not enough?

Reply:
It may not enough for 30 ensembles but this section is for the RRFS with 5 ensembles only.